src/services/cifSemanticValidationService.js

```diff
@@ -0,0 +1,52 @@
+services.factory('cifSemanticValidationService', [function(){
+   function validationSemantic(cif){
+		//In valueCif we have the cif number without the first and last characters
```

Comment:
I see this function too long; I don´t know, maybe it can be refactored into other little pieces.